firefox_puppeteer/ui/toolbars.py

```diff
@@ -343,6 +343,25 @@ def selected_index(self):
 class IdentityPopup(BaseLib):
     """Wraps DOM elements and methods for interacting with the identity popup."""
 
+    def close(self, force=False):
+        """Closes the identity popup.
+
+        :param force: If true, the popup is closed by its own hide function,
+                      otherwise a key event is sent to close the popup.
+        """
+        if not self.is_open:
+            return
+
+        if force:
+            self.marionette.execute_script("""
+              arguments[0].hidePopup();
+            """, script_args=[self.popup])
+        else:
+            (self.marionette.find_element(By.ID, 'identity-popup')
+                            .send_keys(Keys.ESCAPE))
+        Wait(self.marionette).until(
+            lambda _: not self.is_open)
```

Comment:
Please put this all in a single line. There is enough free space for it. Also please add a blank line above to make it more obvious that it doesn't belong to the else case.